modules/serverless-upgrade-sub-channel.adoc

```diff
@@ -5,25 +5,26 @@
 [id="serverless-upgrade-sub-channel_{context}"]
 = Upgrading the Subscription Channel
 
-To upgrade to the latest version of {ServerlessProductName} on {product-title} 4.4, you must update the channel to `4.4`.
+To upgrade to the latest version of {ServerlessProductName} on {product-title} 4.4, you must update the channel to *4.4*.
```

Comment:
This was probably right the way it was. Because it's still just a literal value and not something in the GUI yet.